src/test/c_api_tests.cpp

```diff
@@ -824,9 +828,35 @@ class CAPIState : public ::testing::Test {
     };
 };
 
+TEST_F(CAPIState, ServerLive) {
```

Comment:
We should have a test checking that server is not live and not ready before calling OVMS_StartFromConfigurationFile, and that it is in fact live and ready after the call. We could also check what happens in case of corrupted config file.